frontend/lib/resolution/Resolver.cpp

```diff
@@ -1847,9 +1891,10 @@ Resolver::issueErrorForFailedCallResolution(const uast::AstNode* astForErr,
       context->error(astForErr, "Cannot resolve call to '%s': ambiguity",
                      ci.name().c_str());
     } else {
+      std::vector<const uast::VarLikeDecl*> uninitializedActuals;
       // could not find a most specific candidate
       std::vector<ApplicabilityResult> rejected;
-      CHPL_REPORT(context, NoMatchingCandidates, astForErr, ci, rejected);
+      CHPL_REPORT(context, NoMatchingCandidates, astForErr, ci, rejected, uninitializedActuals);
```

Comment:
Consider the same assertion:

```
CHPL_ASSERT(rejected.size() == actualDecls.size());
```

above. It trivially holds now, but what about the future?

Reply:
That is what I'm suggesting, yep.